pages/_error.tsx

```diff
@@ -41,9 +41,9 @@ const CustomErrorPage: NextPage<CustomErrorPageProps> = ({ title, detail }) => {
   );
 };
 
-CustomErrorPage.getInitialProps = async (contextData) => {
-  return NextErrorComponent.getInitialProps(contextData);
-};
+// CustomErrorPage.getInitialProps = async (contextData) => {
+//   return NextErrorComponent.getInitialProps(contextData);
+// };
```

Comment:
I am not sure what this does exactly, does it pass `title` and `detail` from context? ? If yes, we can create an issue to bring this back in a proper way.

Reply:
agreed, see: https://github.com/acikkaynak/deprem-yardim-frontend/pull/744#issuecomment-1426675453